HW11/IhorBohush/11-1_Practical_task_1_Bohush.py

```diff
@@ -0,0 +1,24 @@
+def age_status(age):
+    """This function processes the information to check is the user's age even or odd number.
+    If a number of the user's age is negative it raises an exception"""
+    try:
+        age = int(age)
+        if age <= 0:
+            raise ValueError(f'that is not a positive number: {age}')
+        elif age % 2 == 0:
+            return 'Your age is an even number'
+        else:
+            return 'Your age is an odd number'
+    except ValueError as e:
```

Comment:
There's no need to raise an Error and catch it later. You can just return ValueError(f'that is not a positive number: {age}')